New title AWS compute services & improved wording

aleon1220 opened the pull request:
i moved the new title AWS Compute with the Spinnaker AWS Cloud Provider to the top
i added a link to an official AWS lambda and spinnaker
this sets the stage for new Spinnaker docs on AWS Lambda
the concepts block is at the bottom.
In a future PR i will improve the wording of concepts as it requires some more detail

aleon1220 committed:
New title AWS compute services & improved wording

i moved the new title AWS Compute with the Spinnaker AWS Cloud Provider to the top
i added a link to an official AWS lambda and spinnaker
this sets the stage for new Spinnaker docs on AWS Lambda
the concepts block is at the bottom.
In a future PR i will improve the wording of concepts as it requires some more detail

aimeeu commented:
See Slack [thread](https://spinnakerteam.slack.com/archives/CMPS49682/p1595197821280500) for details

dorbin commented on setup/install/providers/aws/index.md:
```diff
@@ -7,6 +7,16 @@ sidebar:
 {% include toc %}
 
+### AWS Compute with the Spinnaker AWS Cloud Provider
+The AWS Cloud Provider allows Spinnaker to release artifacts in some of the [AWS compute services](https://aws.amazon.com/products/compute/) with Spinnaker.
```
"allows Spinnaker" and "with Spinnaker" seem redundant.

aleon1220 replied:
i will chage the wording there.

dorbin commented on setup/install/providers/aws/index.md:
```diff
@@ -7,6 +7,16 @@ sidebar:
 
 {% include toc %}
 
+### AWS Compute with the Spinnaker AWS Cloud Provider
```
I think it's a good idea to omit the heading at the beginning of the article (though we don't follow that consistently in this doc set). Is this heading necessary, given that the instructions are meant to be just how to set up an AWS provider?

dorbin replied:
If you decide this heading is worth keeping, promote it to heading level 2.

aleon1220 replied:
mmm yes maybe is best to remove that heading. Gonna do it then from my PR.

dorbin commented on setup/install/providers/aws/index.md:
```diff
@@ -7,6 +7,16 @@ sidebar:
+* [Amazon Elastic Kubernetes Service (EKS)](/setup/install/providers/kubernetes-v2/aws-eks/) - Use this option, if you want to manage containers in [AWS EKS](https://aws.amazon.com/eks/). This option uses [Kubernetes V2 (manifest based) Clouddriver](/setup/install/providers/kubernetes-v2)
+* [Amazon Lambda (Lambda)](https://aws.amazon.com/blogs/opensource/how-to-integrate-aws-lambda-with-spinnaker/) - Use this option, if you want to enable [AWS Lambda](https://aws.amazon.com/lambda/) support 
+
 ### Concepts
```
Beyond the scope of your changes, so feel free to ignore, but...

This was already a level-3 heading before you got here, @aleon1220 , but I'm not sure why. Can you promote to h2?

Also, "concepts" is pretty vague, considering this section is recognizably about AWS accounts. Maybe change it to "About AWS accounts"?

aleon1220 replied:
OK. I will promote it to h2.
About the concepts, i agree with you is vague. That section relates to how Spinnaker uses AWS IAM to have full permissions to deploy and manage apps in AWS. I could think of a title involving permissions or Identity Access Management.That for a future PR.
Thanks

dorbin reviewed (commented):
Thanks for the PR, @aleon1220 ! Just a few comments.

aleon1220 committed:
Merge branch 'master' into patch-2

aleon1220 committed:
Changes suggested and increased headers level

I added the changes suggested by @dorbin
I changed the title of concepts to AWS IAM Permissions with the AWS Cloud Provider
Hopefully that explains  the whole section better
I added a line describing why is AWS IAM required for the spinnaker AWS Cloud provider to work properly.

aleon1220 reviewed (commented):
@dorbin I have added the improvements and a line to explain the AWS Accounts section.

dorbin committed:
Merge branch 'master' into patch-2
